src/styles/order-service.less

```diff
@@ -52,17 +62,13 @@
       padding-top: 8px;
     }
   }
-  .order-service-deails-bottom {
-    height: 30%;
-    padding-left: 180px;
-    padding-right: 180px;
-
-    .wizard-pf-steps-indicator .visited .wizard-pf-step-number {
-      cursor: pointer;
-      color: @color-pf-black;
-      background-color: @color-pf-white;
-      border-color: @color-pf-blue-300;
-    }
+  .order-service-description {
+    clear: both;
+    margin-top: 15px;
+  }
+  .tag {
+    margin-right: 5px;
```

Comment:
should this be on a .tag + .tag selector as a margin-left so we dont have a hanging 5px margin on the last one